### ISSUE TITLE
Clarification: APAC being adopted for WCAG 2.x?

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#2099]
I want to ask about [this tweet](https://twitter.com/MyndexResearch/status/1450688289317425159?s=20) from WCAG 3 co-author @myndex ([screenshot of the whole thread](https://cln.sh/yUGNiZQ9K1JY4gbeboeM)). It (partly) reads:

> The APCA is a new contrast method developed for WCAG 3 to replace the much maligned WCAG 2 1.4.3 contrast method.

I [asked for clarification](https://twitter.com/yatil/status/1450693276579864578) if there are plans to “replace the much maligned WCAG 2 1.4.3 contrast method”. I had understood that WCAG 3 would eventually replace WCAG 2. While I think that this is meant here, I think it can be easily misunderstood as APCA replacing the WCAG 2 method in some iteration of WCAG 2. I don’t think there are any plans for that, correct?

There was no reply to my question (despite sending other tweets), so I thought I am asking here for clarification.

**Comment by mraccess77** [comment]
APCA is currently not referenced in any WCAG 2.x recommendations or informative documents.

WCAG 3.0 is still being developed and specific aspects related to contrast are not finalized.

WCAG 2.x needs to be backwards compatible with WCAG 2.0 and thus anything that fails or passes current SC related to contrast would continue to do so in WCAG 2.1, 2.2, etc.  So there is no change to contrast requirements for WCAG 2.1 or 2.2.

**Comment by yatil** [comment]
Thanks for the confirmation, @mraccess77. That was my understanding, too.
